kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/SerializationProperties.java

```diff
@@ -37,7 +37,7 @@ public class SerializationProperties {
      *
      * @return configuration for transient fields handling.
      */
-    public TransientsProperties getTransients() {
+    TransientsProperties getTransients() {
```

Comment:
Are we verifying that this is still discovered by Spring and reflected on the metadata for `.properties` files?